pootle/static/js/helpers.js

```diff
@@ -37,10 +37,13 @@ var helpers = {
         bodyPadding = parseInt($body.css('padding-bottom'), 10),
         contentAreaHeight = $('#wrapper').height() - $body.offset().top -
                             bodyPadding,
-        sidebarHeight = $('#sidebar #sidebar-content').height() +
-                        $('#footer').height() + bodyPadding,
-        newHeight = Math.max(contentAreaHeight, sidebarHeight);
+        sidebarHeight, newHeight;
 
+    $('#sidebar').css("width", "30%");
```

Comment:
It's [the width of the sidebar](https://github.com/translate/pootle/blob/a328894179ff7c7a4ae6f88bf15087f979590865/pootle/static/css/style.css#L531) when it's open.